doc/libpmem2/pmem2_config_set_address.3.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEM2_CONFIG_SET_ADDRESS, 3)
+collection: libpmem2
+header: PMDK
+date: pmem2 API version 1.0
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2019, Intel Corporation)
+
+[comment]: <> (pmem2_config_set_address.3 -- man page for libpmem2 config API)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**pmem2_config_set_address**() - set address among with address flag type in the
+pmem2_config structure
+
+# SYNOPSIS #
+
+```c
+#include <libpmem2.h>
+
+struct pmem2_config;
+int pmem2_config_set_address(struct pmem2_config *cfg, void *addr, int type);
+```
+
+# DESCRIPTION #
+
+The **pmem2_config_set_address**() function configures the address among with address flag
+which will be used for mapping. It takes one flag type only. *\*config* should be already
+initialized, please see **pmem2_config_new**(3) for details. The *\addr* cannot be NULL and
+must be a multiple of the alignment required for the data source which will be used for
+mapping alongside the config. To retrieve the alignment required for specific instance of
+**pmem2_source** use **pmem2_source_alignment**(3).
+
+Possible flags are:
+
+* **PMEM2_ADDRESS_FIXED_NOREPLACE** - Try to place the mapping at exactly that address
+which was set. When address is occupied, it never clobbers a preexisting mapped range and
+**pmem2_map**(3) returns **PMEM2_E_MAPPING_EXISTS**.
+Note: for Linux it is supported science Linux 4.17.
```

Comment:
1. add `>` (sign for Notes we are using in .md file) 
2. explain why (4.17) and what happens otherwise.